### DISCUSSION TITLE
[doc] Blazor Components Theme like Radzen on Figma

### DISCUSSION THREAD

**Discussion by q0v0p** [havit/Havit.Blazor#681] [General]
If there is a component design theme like Radzen Blazor, I think Havit Blazor will be easier to use in collaboration with the designer.
If you have any, please share them.

https://www.figma.com/community/file/1045331267659707061/blazor-components-standard-theme

**Comment by crdo** [comment]
Yes, it would be really handy, but we do not have it and it is currently out of our scope.
I usually use some of the Bootstrap Figma UI kits that is out there.
If I need something Havit.Blazor specific, I usually use HTML to Figma plugin that I run against our docs that generates the figma components automatically.
It is not perfect structure-wise, but it works.

**Comment by q0v0p** [comment]
I see, I understand. For now, I'll try the HTML to Figma plugin.
It's a great OSS, so I'll continue to support it.
I wish you the best. thank you.
